### PR TITLE
changed or marked references to SAG cloud

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/get-familiar-with-the-ui/gui-features.md [hunk 1]
```diff
@@ -76,8 +76,6 @@ The application switcher allows you to quickly switch between applications. Clic
 
 The application switcher shows all {{< product-c8y-iot >}} applications you currently have access to. These can be [subscribed applications](/standard-tenant/ecosystem/#subscribed-applications) or [custom applications](/standard-tenant/ecosystem/#custom-applications). Just click the icon for the desired application to open it as active application.
 
-If you are using {{< sag-cloud >}}, the application switcher also shows other {{< sag-cloud >}} applications at the first level, followed by the {{< product-c8y-iot >}} applications.
```
Comment (BeateRixen) [inline content/get-familiar-with-the-ui/gui-features.md:79]: @janhommes Can this sentence be removed?

File: content/sensor-app/installing-the-sensor-app.md [hunk 2]
```diff
@@ -18,10 +18,10 @@ Using these links from your smartphone takes you to the app's install page:
 Alternatively, you can open the Apple App Store or Google Play Store from your smartphone, search for **{{< sensor-app >}}** and install the app.
 
 You need a {{< product-c8y-iot >}} account to send measurements to the platform.
-If you don't have one yet, you can create a free trial account at [{{< sag-cloud >}}]({{< link-sag-cloud >}}).
+*** If you don't have one yet, you can create a free trial account at [{{< sag-cloud >}}]({{< link-sag-cloud >}}).
```
Comment (BeateRixen) [inline content/sensor-app/installing-the-sensor-app.md]: @rahultsag @gapa-c8y How should we rephrase this sentence to get rid of Software AG cloud here?

File: content/sensor-app/sensorapp-introduction.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The {{< sensor-app >}} is a free smartphone application available for iOS and An
 
 The app is designed to collect measurements from your smartphone, nearby Bluetooth device sensors, and vehicle On-board Debug (OBD) sensors, and send them to the {{< product-c8y-iot >}} platform. It has a straightforward registration workflow to get you up and running quickly and provides an easy way to get data into {{< product-c8y-iot >}}.
 
-If you are using the {{< product-c8y-iot >}} free trial available via {{< sag-cloud >}}, the app ideally helps you to quickly get familiar with the platform as it provides an easy way to connect devices and sensors. Capturing data from Bluetooth devices with {{< product-c8y-iot >}} moreover saves a lot of implementation effort.
+*** If you are using the {{< product-c8y-iot >}} free trial available via {{< sag-cloud >}}, the app ideally helps you to quickly get familiar with the platform as it provides an easy way to connect devices and sensors. Capturing data from Bluetooth devices with {{< product-c8y-iot >}} moreover saves a lot of implementation effort.
```
Comment (BeateRixen) [inline content/sensor-app/sensorapp-introduction.md]: @gapa-c8y @rahultsag How should we rephrase this sentence to get rid of Software AG cloud here?

File: content/standard-tenant/managing-users.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ On tenant creation, there are default roles available that can be used as a samp
 Note that when subscribed to the "feature-user-hierachy" application, the CREATE permission allows to manage (display, create, edit, delete, disable/enable, delegate, manage permissions) underlying users. For details see [Managing user hierarchies](/enterprise-tenant/managing-user-hierarchies).
 {{< /c8y-admon-req >}}
 
-If your tenant is configured for using single sign-on (SSO) in {{< sag-cloud >}}, new users should be created under **My Cloud**, accessible through the application switcher in the upper right corner, so that they are able to use the single sign-on feature.
+** If your tenant is configured for using single sign-on (SSO) in {{< sag-cloud >}}, new users should be created under **My Cloud**, accessible through the application switcher in the upper right corner, so that they are able to use the single sign-on feature.
```
Comment (BeateRixen) [inline content/standard-tenant/managing-users.md]: @JanePorter Can we remove this sentence?  I assume that My Cloud is no longer available?

File: content/get-familiar-with-the-ui/gui-features.md (hunk 1 above)
Comment (NRusenova) [reply content/get-familiar-with-the-ui/gui-features.md:79]: yes, we don't give access to the SAG Cloud anymore, @BeateRixen we can remove it.

File: content/sensor-app/installing-the-sensor-app.md (hunk 2 above)
Comment (NRusenova) [reply content/sensor-app/installing-the-sensor-app.md]: @BeateRixen free trial account can be created from here: https://cumulocity.com/pages/free-trial/

File: content/sensor-app/sensorapp-introduction.md (hunk 3 above)
Comment (NRusenova) [reply content/sensor-app/sensorapp-introduction.md]: @BeateRixen If you are using the {{< product-c8y-iot >}} free trial provides an easy way to connect devices and sensors. Capturing data from Bluetooth devices with {{< product-c8y-iot >}} moreover saves a lot of implementation effort.

File: content/standard-tenant/managing-users.md (hunk 4 above)
Comment (NRusenova) [reply content/standard-tenant/managing-users.md]: @BeateRixen true, we can remove this.

File: content/sensor-app/installing-the-sensor-app.md (hunk 2 above)
Comment (BeateRixen) [reply content/sensor-app/installing-the-sensor-app.md]: Changed

File: content/standard-tenant/managing-users.md (hunk 4 above)
Comment (BeateRixen) [reply content/standard-tenant/managing-users.md]: Done

File: content/sensor-app/sensorapp-introduction.md (hunk 3 above)
Comment (BeateRixen) [reply content/sensor-app/sensorapp-introduction.md]: @NRusenova Not sure if I got it right, your sentence is missing something. Please check ones more.

File: content/get-familiar-with-the-ui/gui-features.md (hunk 1 above)
Comment (BeateRixen) [reply content/get-familiar-with-the-ui/gui-features.md:79]: Done

File: content/sensor-app/installing-the-sensor-app.md [hunk 5]
```diff
@@ -18,10 +18,10 @@ Using these links from your smartphone takes you to the app's install page:
 Alternatively, you can open the Apple App Store or Google Play Store from your smartphone, search for **{{< sensor-app >}}** and install the app.
 
 You need a {{< product-c8y-iot >}} account to send measurements to the platform.
-If you don't have one yet, you can create a free trial account at [{{< sag-cloud >}}]({{< link-sag-cloud >}}).
+If you don't have one yet, you can create a free trial account from [https://{{< domain-c8y >}}/pages/free-trial/](from here: https://{{< domain-c8y >}}/pages/free-trial/).
 Click the **Login** button at the top right of the page and follow the instructions to create a new account.
 There are no limitations within a {{< product-c8y-iot >}} free trial.
```
Comment (BeateRixen) [inline content/sensor-app/installing-the-sensor-app.md:23]: @NRusenova Is this sentence actually really true?

Comment (NRusenova) [reply content/sensor-app/installing-the-sensor-app.md:23]: If you don't have one yet, you can create a free trial account at https://cumulocity.com/pages/free-trial/

File: content/sensor-app/sensorapp-introduction.md (hunk 3 above)
Comment (NRusenova) [reply content/sensor-app/sensorapp-introduction.md]: The free trial provides an easy way to connect devices and sensors, capturing data from Bluetooth devices with {{< product-c8y-iot >}} moreover saves a lot of implementation effort.

File: content/sensor-app/installing-the-sensor-app.md (hunk 5 above)
Comment (NRusenova) [reply content/sensor-app/installing-the-sensor-app.md:23]: @BeateRixen I've updated the sentence.

File: content/sensor-app/sensorapp-introduction.md (hunk 3 above)
Comment (NRusenova) [reply content/sensor-app/sensorapp-introduction.md]: @BeateRixen I think we can just remove the whole sentence, it duplicates the text above.

**Review by MWindel — APPROVED** [review]

**Pull request merged**